code/game/gamemodes/cult/narsie.dm

```diff
@@ -95,12 +95,13 @@ GLOBAL_LIST_EMPTY(narsie_list)
 	else if(istype(A, /obj/structure/cult))
 		qdel(A)
 
-/obj/singularity/narsie/large/CollidedWith(atom/A)
-	if(!cause_hell) return
-	if(isturf(A))
-		narsiewall(A)
```

Comment:
This is likely to change the function of narsie.


To restore that you need to retype collided with to atoms (from atom movable)

What is the reason for using atom/movable instead of atom as parameter?